main.go

```diff
@@ -184,3 +188,19 @@ func handleForceClose() {
 
 	launchTUI()
 }
+
+func handleOutputAll(format string) {
+	items := config.TextItems()
+
+	if format == "raw" {
+		for _, v := range items {
+			fmt.Printf("%q\n", v.Value)
+		}
+	} else if format == "unescaped" {
+		for _, v := range items {
+			fmt.Println(v.Value)
+		}
+	} else {
+		fmt.Printf("Invalid argument to -output-all\nSee %s --help for usage", os.Args[0])
+	}
```

Comment:
This could be a `switch format `

Reply:
@wassup05 As we wouldn't expect further options here, I'm wondering if there's an elegant way to drop the need for specifying the "unescaped" value and use that as the default if the `-output-all` flag is supplied, with the option to add "raw" if needed. Just thinking that could be a simpler implementation for both the UX and the code. 

I realise this isn't possible with a string flag in Go, as a value would always be needed there, but maybe setting this to a bool and checking for the "raw" arg explicitly could be a simple way to achieve this. What do you think?

Reply:
E.g. 

```go

outputAll   = flag.Bool("output-all", false, "Print clipboard text content to stdout, each entry separated by a newline. Add the value 'raw' for escaped output.")
extraArgVal = flag.Arg(2)

...

case *outputAll:
	handleOutputAll()

...

func handleOutputAll() {
	items := config.TextItems()

	if extraArgVal == "raw" {
		for _, v := range items {
			fmt.Printf("%q\n", v.Value)
		}
		return
	}

	for _, v := range items {
		fmt.Println(v.Value)
	}
}
```

Reply:
Hi @savedra1, Thanks for reviewing
Although I see how that would be useful, It could bring some inconsistencies into pitcure like `flag` supports 4 formats iirc,

`-flag a`
`-flag=a`
`--flag a`
`--flag=a`

But we wouldn't be able to do that if we make it a `flag.Bool`

Another alternative would be to set the `flag.String` default value as something different from "raw" or "unescaped" like "notset" and then we see if this has been changed by the user... But I see a drawback with this too which being that the user may be misguided by the default value printed by `flag.PrintDefaults`

What do you think?

Reply:
Hi @wassup05, apologies for such a delayed response here. 

I take your point, I think your approach works well as it is, so there's no need to overthink. 

I will be merging this now 👍 

Thank you again for your work here.